topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -0,0 +1,1385 @@
+>    - Click on the newly generated FastQ pair ending in *"with matched ID"* to expand the individual reads
+>      - Click on the {% icon galaxy-eye %} symbol of the forward read
+>      - Click somewhere outside the white box to close the **Scratchbook**
+>      - Click on the {% icon galaxy-eye %} symbol of the reverse read
+>    - Position/Resize the boxes as desired
+{: .hands_on}
+
+<!-- 
+These are reads that all map to ENSDARG00000019692. In [Cell, UMI] format:
+
+(J00182:75:HTKJNBBXX:2:1114:12469:11073|J00182:75:HTKJNBBXX:2:2222:13301:35690|J00182:75:HTKJNBBXX:2:1203:25022:13763|J00182:75:HTKJNBBXX:2:1115:8501:46961)
+
+            Cell  , UMI
+1: 46961 -- ACCAGA, GGAAGA
+2: 13763 -- GGTAAC, GTCCCA -> same umi, same cell
+3: 35690 -- GGTAAC, GTCCCA -> same umi, same cell
+4: 11073 -- GGTAAC, CGGCGT -> diff umi, same cell
+-->
+
+### Our Four Reads of Interest
+
+Let us examine these reads four reads of interest which we have just sub-selected using their headers:
+
+> ### {% icon details %} Reads:
+>
+> Forward:
+>
+>         @J00182:75:HTKJNBBXX:2:1115:8501:46961 1:N:0:ATCACG
+>         GGAAGAACCAGATTTTTTTTTTTTTTTTTT
+>         +
+>         AAFFFJJJJJJJFFFJJJJJJJJJJJJJJJ
+>         
+>         @J00182:75:HTKJNBBXX:2:1203:25022:13763 1:N:0:ATCACG
+>         GTCCCAGGTAACTTTTTTTTTTTTTTTTTT
+>         +
+>         AAFFFJJJJJJJJFFJJJJJJJJJFJ<FF-
+>         
+>         @J00182:75:HTKJNBBXX:2:2222:13301:35690 1:N:0:ATCACG
+>         GTCCCAGGTAACTTTTTTTTTTTTTTTTTT
+>         +
+>         AAFFFJJJJJJJ<AFJJJJJFFJJFJJJFF
+>         
+>         @J00182:75:HTKJNBBXX:2:1114:12469:11073 1:N:0:ATCACG
+>         CGGCGTGGTAACTTTTTTTTTTTTTTTTCC
+>         +
+>         AAFFFJJJJJJJFAFFJJJJJJJJF---<F
+>         
+> 
+> Reverse:
+> 
+>         @J00182:75:HTKJNBBXX:2:1115:8501:46961 2:N:0:ATCACG
+>         GACCTCTGATCTTTACGAAAGGCCAACGCGTTTTCAGTCTGGACACGGTTCAGCTCCTGTTCATTATTCA
+>         +
+>         A<<A-777F<AA<AJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ
+>         
+>         @J00182:75:HTKJNBBXX:2:1203:25022:13763 2:N:0:ATCACG
+>         GCCACCTAATTTCCGTCATCACACTCCTCTCCGTTTTCAACTTGCACAATGCTGTCTCCGCAGAATCCCT
+>         +
+>         ---<----<A---77-7A-FJ<JJFFJJ<JJAJ7<-FAFFJJFF<FFJJFFAJFA-AFFFJFFFFFJAJJ
+>         
+>         @J00182:75:HTKJNBBXX:2:2222:13301:35690 2:N:0:ATCACG
+>         CAATCCTCTCCGTTATCAACTTGCACAATGCTGTCTCCGCAGAATCCCTCCGGATCAGGATCGCTCTCCA
+>         +
+>         <<A-77--77F<----7AFJ-A--FJJJFAJF-AFAJAJ<JFJ<JJJFFJJJFJJJJJAAFJJJFJJJF-
+>         
+>         @J00182:75:HTKJNBBXX:2:1114:12469:11073 2:N:0:ATCACG
+>         ATCCACTTATTGCAAAGCAGAGGACATTGAGTCTCACCTTTTGTCCAGGTCTTCCAATTTCACCCTGCAA
+>         +
+>         A-77AA-7FF<7FFJFFFJJJJJJJJJJJJJ-AFJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJ
+>         
+{: .details}
+
+What we observe are the standard four lines of any FASTQ file:
+  1. Read name starting with `@`
+  2. Sequence of nucleotide bases
+  3. Seperator `+`
+  4. Quality string of the nucleotide bases in Hex
```

Comment:
this is wrong: if anything then it's ASCII, but the question is whether that's worth mentioning here. Maybe just quality string is good enough?

Reply:
changed it to ASCII, leaving it as quality string is just too vague I think